src/dawn_native/d3d12/DeviceD3D12.h

```diff
@@ -78,6 +77,7 @@ namespace dawn_native { namespace d3d12 {
         ResultOrError<BindGroupLayoutBase*> CreateBindGroupLayoutImpl(
             const BindGroupLayoutDescriptor* descriptor) override;
         ResultOrError<BufferBase*> CreateBufferImpl(const BufferDescriptor* descriptor) override;
+        ResultOrError<TextureBase*> CreateTextureImpl(const TextureDescriptor* descriptor) override;
```

Comment:
nit: sort, here and other DeviceXXX and in NullBackend too

Reply:
Done